runtime/common/src/parachains.rs

```diff
@@ -1425,6 +1446,13 @@ impl<T> sp_std::fmt::Debug for ValidateDoubleVoteReports<T> where
 	}
 }
 
+impl<T> ValidateDoubleVoteReports<T> {
+	/// Create a new `ValidateDoubleVoteReports` struct.
+	pub fn new() -> Self {
```

Comment:
It's mostly a matter of opinion so feel free to ignore it but why not implement `Default` instead?